mailbox/elasticsearch/pom.xml

```diff
@@ -89,6 +89,12 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-guice-common</artifactId>
```

Comment:
Why is this needed?

This dependency feels bad...

Reply:
Not sure myself, but the compiler wouldn't stop crying that it's missing this overwise since I switched to the `TikaExtension` that is using `GuiceModuleTestExtension`... But yes there is probably better to do here than just the copy/paste of something existing